homeworkW12D03/src/ba/bitcamp/homeworkW12D03/task01/CountLettersFromFile.java

```diff
@@ -0,0 +1,139 @@
+package ba.bitcamp.homeworkW12D03.task01;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.concurrent.LinkedBlockingQueue;
+
+/**
+ * 
+ * @author Gordan
+ *
+ *         This class represents one program that counts how many time first
+ *         letter from text file repeats in same text file.
+ */
+public class CountLettersFromFile {
+
+	// Defining variables
+	private static int counter = 1;
+	private static LinkedBlockingQueue<Master> assignments;
+	private static ArrayList<Worker> workers;
+	private static Object lock = new Object();
+
+	public static void main(String[] args) {
+
+		// Initializing list and queue
+		assignments = new LinkedBlockingQueue<>();
+		workers = new ArrayList<>();
+
+		try {
+
+			// Creating reader for reading text file
+			BufferedReader reader = new BufferedReader(new FileReader(new File("Text.txt")));
+			// Taking first letter from file
+			char firstChar = (char) reader.read();
+			// Putting letter in low case
+			firstChar = Character.toLowerCase(firstChar);
+			// Printing first letter
+			System.out.println(firstChar);
+
+			String line = "";
+
+			// Reading the rest of the text
+			while (reader.ready()) {
+				// Taking line and making it low case
+				line = reader.readLine().toLowerCase();
+				// Making master thread
+				Master master = new Master(line, firstChar);
+				// Adding master thread to queue of threads
+				assignments.add(master);
+			}
+
+			reader.close();
```

Comment:
In case reading the file produces an exception, reader is never closed. Always close readers in `finally` block or create them using `try (create reader) {}...`